Orange/widgets/tests/test_owselectcolumns.py

```diff
@@ -105,3 +107,33 @@ def retrieveSpecificSettings(self):
 
     def storeSpecificSettings(self):
         pass
+
+class TestOWSelectAttributes(WidgetTest):
+    def setUp(self):
+        self.widget = self.create_widget(OWSelectAttributes)
+
+    def testall(self, location):
```

Comment:
Methods that group several assertions are usually called assertSomething. In this case, I would go with
assertVariableCountsEqual(available, used, class).

This method would be called below, possibly even with keyword arguments to improve clarity.